microsoft-edge/webview2/how-to/webdriver.md

```diff
@@ -229,6 +229,11 @@ For more info about the `DebuggerAddress` property of the `EdgeOptions` object,
 
 Congratulations!  You've successfully automated a WebView2 project and driven WebView2 by using Selenium and Microsoft Edge WebDriver, by attaching Microsoft Edge WebDriver to a running WebView2 app.
 
+#### UWP projects and debugging
+
```

Comment:
This is a fresh comment, to see latest surrounding code.

Only the regkey approach applies, not the env var approach.  Reg key:

HKEY_CURRENT_USER\Software\Policies\Microsoft\edge\WebView2\AdditionalBrowserArguments
with the assigned value in the "*" REG_SZ. As per our docs.

[Outdated comment](https://github.com/MicrosoftDocs/edge-developer/pull/2228/files/ba60e79deff7841b93c4a6ceb8a6adda6550f102#r986298310) thread

Reply:
Assessment: The rewritten paragraph clarifies the previous thread.  You can Resolve this fresh comment thread.

Reply:
Resolving based on Michael's request.